rag/vector_store.py

```
@@ -0,0 +1,432 @@
+    query_embedding: List[float],
+    where: Dict[str, str],
+    n_results: int,
+) -> List[str]:
+    try:
+        results = collection.query(
+            query_embeddings=[query_embedding],
+            n_results=n_results,
+            where=where,
+            include=["documents", "metadatas"],
+        )
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)